src/fillTank.test.js

```diff
@@ -1,11 +1,99 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared ', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should calculate full tank If the amount is not given', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 0,
+      },
+    };
+
+    fillTank(customer, 30);
+    expect(customer.money).toBe(1800);
+    expect(customer.vehicle.fuelRemains).toBe(40);
+  });
+
+  it(
+    'should calculate full tank If the amount is greater than the tank can use',
+    () => {
+      const customer = {
+        money: 3000,
+        vehicle: {
+          maxTankCapacity: 20,
+          fuelRemains: 5,
+        },
+      };
+
+      fillTank(customer, 50, 20);
```

Comment:
The test case should verify that the function only fills the tank to its maximum capacity when the amount exceeds what the tank can hold. Ensure the `fillTank` function is correctly implemented to handle this scenario.